File: android/app/src/main/java/bjc/shoperp/service/restful/DeliveryOutService.java
package bjc.shoperp.service.restful;

import java.util.Date;
import java.util.HashMap;

import bjc.shoperp.domain.DeliveryOut;
import bjc.shoperp.domain.restfulresponse.domainresponse.DeliveryOutCollectionResponse;

/**
 * Created by hth on 2018/2/25.
 */

public class DeliveryOutService extends ServiceBase<DeliveryOut> {

    public DeliveryOutCollectionResponse GetByAll(long shopId, String deliveryCompany, String deliveryNumber, String vendor, String number, Date startTime, Date endTime, int pageIndex, int pageSize) throws Exception {
        HashMap<String, Object> para = new HashMap<>();
        para.put("payType",0);
        para.put("shopId",shopId);
        para.put("deliveryCompany",deliveryCompany);
        para.put("deliveryNumber",deliveryNumber);
        para.put("vendor",vendor);
        para.put("number",number);
        para.put("startTime",startTime);
        para.put("endTime",endTime);
        para.put("pageIndex",pageIndex);
        para.put("pageSize",pageSize);
        return DoPost(DeliveryOutCollectionResponse.class,para,null);
    }
}

File: net/ShopErp.App/Service/Restful/DeliveryOutService.cs
using ShopErp.Domain;
using ShopErp.Domain.RestfulResponse;
using System;

namespace ShopErp.App.Service.Restful
{
    public class DeliveryOutService : ServiceBase<DeliveryOut>
    {
        public DataCollectionResponse<DeliveryOut> GetByAll(PopPayType payType, long shopId, string deliveryCompany,
            string deliveryNumber, string vendor, string number, string shipper, DateTime startTime, DateTime endTime, int pageIndex,
            int pageSize)
        {
            System.Collections.Generic.Dictionary<string, object> para = new System.Collections.Generic.Dictionary<string, object>();
            para["payType"] = payType;
            para["shopId"] = shopId;
            para["deliveryCompany"] = deliveryCompany;
            para["deliveryNumber"] = deliveryNumber;
            para["vendor"] = vendor;
            para["number"] = number;
            para["shipper"] = shipper;
            para["startTime"] = startTime;
            para["endTime"] = endTime;
            para["pageIndex"] = pageIndex;
            para["pageSize"] = pageSize;
            return DoPost<DataCollectionResponse<DeliveryOut>>(para);
        }

        public ResponseBase DeleteOrderDeliveryOut(string deliveryNumber)
        {
            System.Collections.Generic.Dictionary<string, object> para =
                new System.Collections.Generic.Dictionary<string, object>();
            para["deliveryNumber"] = deliveryNumber;
            return DoPost<ResponseBase>(para);
        }
    }
}